### ISSUE TITLE
throw new Error(msgs.join(' ') || 'Unknown error');

### ISSUE THREAD

**Issue by patomation** [bigcommerce/stencil-cli#577]
### Expected behavior
Stencil start should start or give a better error message

### Actual behavior
This unknown error message happens

```
(node:32132) [DEP0022] DeprecationWarning: os.tmpDir() is deprecated. Use os.tmpdir() instead.
/home/patrick/.nvm/versions/node/v10.19.0/lib/node_modules/@bigcommerce/stencil-cli/node_modules/hapi/node_modules/hoek/lib/index.js:730
    throw new Error(msgs.join(' ') || 'Unknown error');
    ^

Error: Unknown handler: directory
    at Object.exports.assert (/home/patrick/.nvm/versions/node/v10.19.0/lib/node_modules/@bigcommerce/stencil-cli/node_modules/hapi/node_modules/hoek/lib/index.js:730:11)
    at Object.exports.defaults (/home/patrick/.nvm/versions/node/v10.19.0/lib/node_modules/@bigcommerce/stencil-cli/node_modules/hapi/lib/handler.js:108:14)
    at new module.exports.internals.Route (/home/patrick/.nvm/versions/node/v10.19.0/lib/node_modules/@bigcommerce/stencil-cli/node_modules/hapi/lib/route.js:55:37)
    at module.exports.internals.Connection.internals.Connection._addRoute (/home/patrick/.nvm/versions/node/v10.19.0/lib/node_modules/@bigcommerce/stencil-cli/node_modules/hapi/lib/connection.js:367:19)
    at module.exports.internals.Connection.internals.Connection._route (/home/patrick/.nvm/versions/node/v10.19.0/lib/node_modules/@bigcommerce/stencil-cli/node_modules/hapi/lib/connection.js:359:18)
    at module.exports.internals.Plugin.internals.Plugin._apply (/home/patrick/.nvm/versions/node/v10.19.0/lib/node_modules/@bigcommerce/stencil-cli/node_modules/hapi/lib/plugin.js:570:14)
    at module.exports.internals.Plugin.internals.Plugin.route (/home/patrick/.nvm/versions/node/v10.19.0/lib/node_modules/@bigcommerce/stencil-cli/node_modules/hapi/lib/plugin.js:540:10)
    at internals.registerRoutes (/home/patrick/.nvm/versions/node/v10.19.0/lib/node_modules/@bigcommerce/stencil-cli/server/plugins/router/router.module.js:42:12)
    at Items.serial (/home/patrick/.nvm/versions/node/v10.19.0/lib/node_modules/@bigcommerce/stencil-cli/node_modules/hapi/lib/server.js:347:18)
    at iterate (/home/patrick/.nvm/versions/node/v10.19.0/lib/node_modules/@bigcommerce/stencil-cli/node_modules/hapi/node_modules/items/lib/index.js:36:13)
    at Object.exports.serial (/home/patrick/.nvm/versions/node/v10.19.0/lib/node_modules/@bigcommerce/stencil-cli/node_modules/hapi/node_modules/items/lib/index.js:39:9)
    at module.exports.internals.Server.internals.Server._invoke (/home/patrick/.nvm/versions/node/v10.19.0/lib/node_modules/@bigcommerce/stencil-cli/node_modules/hapi/lib/server.js:344:11)
    at Items.parallel (/home/patrick/.nvm/versions/node/v10.19.0/lib/node_modules/@bigcommerce/stencil-cli/node_modules/hapi/lib/server.js:229:14)
    at done (/home/patrick/.nvm/versions/node/v10.19.0/lib/node_modules/@bigcommerce/stencil-cli/node_modules/hapi/node_modules/items/lib/index.js:63:25)
    at process.nextTick (/home/patrick/.nvm/versions/node/v10.19.0/lib/node_modules/@bigcommerce/stencil-cli/node_modules/hapi/node_modules/hoek/lib/index.js:848:22)
    at process._tickCallback (internal/process/next_tick.js:61:11)
 ✘  ~/source/cornerstone   customized-for-indie-ella ●  Webpack Bundle Analyzer saved report to /home/patrick/source/cornerstone/assets/dist/report.html
events.js:174
      throw er; // Unhandled 'error' event
      ^

Error [ERR_IPC_CHANNEL_CLOSED]: Channel closed
    at process.target.send (internal/child_process.js:636:16)
    at Watching.webpack.watch [as handler] (/home/patrick/source/cornerstone/stencil.conf.js:43:17)
    at compiler.hooks.done.callAsync (/home/patrick/source/cornerstone/node_modules/webpack/lib/Watching.js:98:9)
    at _err0 (eval at create (/home/patrick/source/cornerstone/node_modules/tapable/lib/HookCodeFactory.js:33:10), <anonymous>:11:1)
    at Immediate.<anonymous> (/home/patrick/source/cornerstone/node_modules/webpack-bundle-analyzer/lib/BundleAnalyzerPlugin.js:79:13)
    at Generator.next (<anonymous>)
    at asyncGeneratorStep (/home/patrick/source/cornerstone/node_modules/webpack-bundle-analyzer/lib/BundleAnalyzerPlugin.js:3:103)
    at _next (/home/patrick/source/cornerstone/node_modules/webpack-bundle-analyzer/lib/BundleAnalyzerPlugin.js:5:194)
    at process._tickCallback (internal/process/next_tick.js:68:7)
Emitted 'error' event at:
    at process.nextTick (internal/child_process.js:640:35)
    at process._tickCallback (internal/process/next_tick.js:61:11)
```

### Steps to reproduce behavior
1. on linux machine use npm 10.16.3 or 10.19.0
2. install stencil cli
3. Pull repo ```https://github.com/patomation/cornerstone/tree/customized-for-indie-ella```
4. npm install
5. Stencil init
6. Stencil start

**Comment by junedkazi** [comment]
@patomation thanks for reporting. I am looking into it.

**Comment by junedkazi** [comment]
Looks like 1.22.0 works fine and something I broke in 1.23.0 release

**Comment by junedkazi** [comment]
The issue should be fixed in 1.23.1.

**Comment by patomation** [comment]
Awesome thankyou!
